### ISSUE TITLE
Big Sur Support

### ISSUE THREAD

**Issue by Russ93** [electron/electron-notarize#63]
Hey there, Everything is working great on Catalina, but for Big Sur it's not accepting the notarization.

**Comment by kkurz-sangoma** [comment]
+1 on this. Not sure what to look at for more information about *what* is failing, all the codesign and stapler commands I've tried appear to show valid notarization, and we receive successful notarization emails.

**Comment by tance77** [comment]
Just wanted to see if anyone else was having issue with notarization and the application wasn't running on Big Sur. Its been a while since there has been a response here.

**Comment by tance77** [comment]
@kkurz-sangoma I was struggling with the same issue. Getting the success email of notarizations. The issue we finally discovered is that one of our external libraries had a hard coded path to some swift libraries. I was buried deep down there we had to right a node.js script to update the path to be relative so it would work on another persons Mac without Gatekeeper giving you the "this app cannot be verified" 


Ultimately we ended up contacting apple about why Gatekeeper was blocking our what seemed to be notarized correctly application. They provided us this [link](https://developer.apple.com/library/archive/technotes/tn2206/_index.html#//apple_ref/doc/uid/DTS40007919-CH1-TNTAG206 as well as a few specific things related to our application.


I would suggest checking to make sure you have no absolute paths anywhere. 

I hope this helps. Our app is now notarized correctly and Gatekeeper lets it through.
